CONTRIBUTING.md

```diff
@@ -1,8 +1,56 @@
 # Contributing
 
-Thank you for wanting to contribute back to the Sensu Community repository. Here are a few guidelines to follow as you do so: 
+Thank you for wanting to contribute back to the Sensu Community. Here are a few guidelines to follow to open an issue here:
 
 1. All contributors are expected to read and follow [our code of conduct](https://sensuapp.org/conduct). Anyone can reach out to administrators at community@sensu.io if needed.
 2. This repository is designed to be an aggregator of resources for our community. Feel free to extend that idea through new documents or suggestions. We welcome your creativity.
 
+# Contributing to the Plugins & Extensions organizations
+
+We need participation like yours in order to maintain the massive number of plugins and extensions that make Sensu an amazing project. Our ambitious goal is to:
+
+* Have at least 2 organizational-wide maintainers at all times
```

Comment:
I would add that at least one of them must be a community member who is not an employee of sensu inc.

Reply:
Hmm I realized that this gets messy when it comes to `the sensu` org rather than `sensu-plugins` or `sensu-extensions` since an org wide maintainer would have access to private repos and that creates legal issues for anything provided as value add for enterprise. I think the best thing that we can do is call that out explicitly that this applies to the `sensu-plugins` and `sensu-extensions` orgs and that the `sensu` org *open source* projects should have a minimum of 2 maintainers on a per repo basis.

Reply:
OK, then should we remove say the chef, ansible, etc from this doc and have that called out elsewhere?